lib-ee/emqx_ee_connector/rebar.config

```diff
@@ -3,7 +3,8 @@
 {deps, [
   {hstreamdb_erl, {git, "https://github.com/hstreamdb/hstreamdb_erl.git", {tag, "0.2.5"}}},
   {emqx, {path, "../../apps/emqx"}},
-  {emqx_utils, {path, "../../apps/emqx_utils"}}
+  {emqx_utils, {path, "../../apps/emqx_utils"}},
+  {emqx_redis, {path, "../../apps/emqx_redis"}}
```

Comment:
Q: do we need it here now that it was moved to its own app?

Reply:
You are not needed. I did the refactoring in two steps (broke out the open source connector first) and it seems like I forgot to delete this when it was not needed any longer.

Reply:
Fixed in [5de6aed](https://github.com/emqx/emqx/pull/11139/commits/5de6aedc31956b49cc5af1dc5c6f273b8e55b94b)